Gcpe.Hub.API/Gcpe.Hub.API.sln

```diff
@@ -0,0 +1,17 @@
+
```

Comment:
I don't think you want this file. There's another one up on level

Reply:
thanks! removed